### PR TITLE
Updated REST project version number

### PR DESCRIPTION
**Pull request by akshelton**
Updated the project version number for the Celigo.ServiceManager.NetSuite.REST project from 1.0.6 to 1.1.0. The version number update was missing from PR [#42 ](url).

### CODE REVIEW

File: Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
       <TargetFramework>netcoreapp3.1</TargetFramework>
-      <Version>1.0.6</Version>
+      <Version>1.1.0</Version>
```
Comment (sameera) [inline Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj:5]: @akshelton Would you mind adding a `PackageReleaseNotes` as well and mention the changes?

Comment (akshelton) [reply Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj:5]: @sameera Thank you for reminding me. I will add a section for `PackageReleaseNotes`.

File: Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj [hunk 2]
```diff
@@ -15,6 +15,9 @@
       <RepositoryType>Git</RepositoryType>
       <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
       <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+      <PackageReleaseNotes>
```
Comment (akshelton) [inline Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj:18]: @sameera Is this a sufficient version release note?

**Review by sameera — APPROVED** [review]

**Comment by sjd2021** [comment]
What needs to happen for this package to be deployed and available in nuget?

**Comment by sjd2021** [comment]
@sameera  Thanks for putting this together! What needs to happen for this package to be deployed and available in nuget?

**Comment by sameera** [comment]
@sjd2021 This is achieved by pushing a release tag which is picked up by the CI/CD pipeline.
I have done this now.
https://www.nuget.org/packages/Celigo.ServiceManager.NetSuite.REST/1.1.0

**Pull request merged**